src/main/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReader.scala

```diff
@@ -0,0 +1,31 @@
+package org.allenai.deep_qa.data
+
+import scala.collection.mutable
+import scala.collection.JavaConverters._
+
+import com.opencsv.CSVReader
+import com.mattg.util.FileUtil
+import java.io.StringReader
+import org.json4s._
+import org.json4s.native.JsonMethods.parse
+
+/*
+ * Read an Omnibus direct answer file pulled from the Aristo
+ * evaluation framework.
+ */
+
+class OmnibusDaDatasetReader(fileUtil: FileUtil) extends DatasetReader[DirectAnswerInstance] {
```

Comment:
Just don't pass it the `FileUtil` object in that code; instantiate it with no parameters.  That should be fine.